llvm/lib/LTO/LTO.cpp

```diff
@@ -121,6 +121,9 @@ void llvm::computeLTOCacheKey(
     support::endian::write64le(Data, I);
     Hasher.update(Data);
   };
+  auto AddUint8 = [&](const uint8_t &I) {
```

Comment:
nit: could pass by value, since uint8_t is small (similar to AddUnsigned)?

Reply:
done.